homeassistant/components/hue/strings.json

```diff
@@ -68,6 +68,11 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "tamper_sensor": {
+        "name": "Tampering"
+      }
```

Comment:
Can we use device class translations for this?

Reply:
Apparently not... There is no device class translation for tampering

Reply:
There is a device class Tamper right?

Reply:
Yeah, but it does not translate the name of the entity... ?

Reply:
let me check

Reply:
Oh lol, it doesn't have translations, we should add them there instead

Reply:
Sounds good, let's fix that first and merge this PR afterwards then.

Reply:
Removed the custom translation_key for the tamper sensor in favor of the generic fix for the Tamper device class translation. Test will now fail until the generic issue is resolved so let's keep this PR draft. Will you fix the Tamper device class thingie or should I ?

Reply:
I will take a look after the stream